config/environments/test.rb

```diff
@@ -39,6 +39,11 @@
   # Print deprecation notices to the stderr.
   config.active_support.deprecation = :stderr
 
+  # Turn off ActiveRecord logging in the test environment
```

Comment:
Just curious: what was this needed for?

Reply:
When I was trying to figure out what was happening in Travis and whenever I ran tests locally, I was getting all of the SQL that was running with all the tests, which was making it super hard to figure out where anything was. Basically just cleans up noise.

Reply:
gotcha